app/controllers/sufia/admin/strategies_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Sufia
+  module Admin
+    class StrategiesController < Flip::StrategiesController
+      before_action do
+        authorize! :manage, Feature
```

Comment:
I just wasn't looking at the one when doing the other. The fully-qualified path is optional. Semantically they are the same.